default/scripting/buildings/PLANET_BEACON.focs.txt

```diff
@@ -19,4 +24,5 @@ BuildingType
     icon = "icons/building/beacon.png"
 
 #include "/scripting/common/enqueue.macros"
-#include "/scripting/common/base_prod.macros"
\ No newline at end of file
+#include "/scripting/common/base_prod.macros"
```

Comment:
is this `#include` needed? it doesn't seem relevant to the script additions.

Reply:
The include was in the file to begin with. I think there was a newline appended causing it to be highlighted? Removed.

Reply:
probably needed then (though ideally without the whitespace noise)